clang/docs/analyzer/checkers.rst

```diff
@@ -804,10 +804,28 @@ Check for performance anti-patterns when using Grand Central Dispatch.
 
 .. _optin-performance-Padding:
 
-optin.performance.Padding
-"""""""""""""""""""""""""
+optin.performance.Padding (PaddingChecker)
```

Comment:
It should be the languages within the parens, that the checker supports.
I think this checker applies for C, C++ and probably even for ObjC as well.

Reply:
Thank you so much for your valuable feedbacks i will update it as you said.